pydy/codegen/code.py

```diff
@@ -3,28 +3,43 @@
 """This module remains for backwards compatibility reasons and will be
 removed in PyDy 0.4.0."""
 
+from pkg_resources import parse_version
 import warnings
 
+import pydy
+
 from .ode_function_generators import generate_ode_function as new_gen_ode_func
 
-with warnings.catch_warnings():
-    warnings.simplefilter('once')
-    warnings.warn("This module, 'pydy.codgen.code', is deprecated. The "
-                  "function 'generate_ode_function' can be found in the "
-                  "'pydy.codegen.ode_function_generator' module. "
-                  "'CythonGenerator' has been removed, use "
-                  "'pydy.codegen.cython_code.CythonMatrixGenerator' "
-                  "instead.",
-                  DeprecationWarning)
+
+class PyDyCodeGenCodeDeprecationWarning(DeprecationWarning):
+        pass
+class GenerateOdeFunctionDeprecationWarning(DeprecationWarning):
+        pass
+class CythonGeneratorDeprecationWarning(DeprecationWarning):
+        pass
+warnings.simplefilter('once', PyDyCodeGenCodeDeprecationWarning)
```

Comment:
I'm going to do a PyDyDepWarning that inherits from DeprecationWarning as someone might want to ignore Sympy warnings without know it will also filter PyDy ones.